crates/accelerate/README.md

```diff
@@ -8,3 +8,13 @@ place to put the code.  This is _usually_ the right place to put new Rust/Python
 The `qiskit-pyext` crate is what actually builds the C extension modules.  Modules in here should define
 themselves has being submodules of `qiskit._accelerate`, and then the `qiskit-pyext` crate should bind them
 into its `fn _accelerate` when it's making the C extension.
+
+Specifically, adding a rust module `my_module` to Qiskit such that it can be called from within Python involves
+the following steps:
+1. Add `pub fn my_module(m: &Bound<PyModule>) -> PyResult<()>` to your `my_module.rs` file with `m.add_wrapped(wrap_pyfunction!(mymodulefunction))?;` for new functions and `m.add_class::<MyModuleClass>()?;` for new classes.
```

Comment:
This could work better with a code block:
```suggestion
1. Add a module function to your to your `my_module.rs` file, it should look similar to this:
  
     ```rust
     pub fn my_module(m: &Bound<PyModule>) -> PyResult<()> {}
     ```
     The body of this function should add each class/method/function that you want to be exposed to python through this module. It should look similar to this:
     ```rust
     pub fn my_module(m: &Bound<PyModule>) -> PyResult<()> {
        m.add_wrapped(wrap_pyfunction!(mymodulefunction))?; // For functions.
        m.add_class::<MyModuleClass>()?; // For classes.
     }
     ```
```